README.md

```diff
@@ -110,9 +124,12 @@ Let's create an MTC CA.
 $ mtc ca new --batch-duration 5m --lifetime 1h 62253.12.15 ca.example.com/path
 ```
 
-This creates a new MTC CA called `my-mtc-ca`, and puts the data in the
-current working directory. A batch is issued every 5 minutes, and
-each batch is valid for one hour.
+This creates a new MTC CA in the current working directory. It's configured
+to issue a batch every 5 minutes, and for each batch to be valid for an hour.
```

Comment:
We should note here that 5 minutes is smaller than the batch duration we expect to see in prod (~hour?)

Reply:
Done